src/corona_hakab_model/generation/matrix_consts.py

```diff
@@ -86,6 +88,12 @@ def from_file(cls, param_path):
 
         return cls(**parameters)
 
+    def export(self, export_path, file_name: str = "matrix_consts.json"):
+        if not file_name.endswith(".json"):
+            file_name += ".json"
+        with open(os.path.join(export_path, file_name), "w") as export_file:
+            export_file.write(jsonpickle.encode(self._asdict()))
```

Comment:
doesnt dict(self) works?

Reply:
no, returns an error. 

we can however do: 
`jsonpickle.endoce(self)`
and encode the namedtuple directly. it would probably be easier to decode straight into a consts object, but it is less readable as a json